components/molecule/tabs/src/components/MoleculeTab.js

```diff
@@ -20,16 +22,28 @@ const MoleculeTab = ({
   label,
   numTab
 }) => {
+  const [isIntersecting, outerRef] = useNearScreen({offset: '0px'})
```

Comment:
`useNearScreen` is a wrapper of useOnScreen with default offset set to 200px. if you want to set if to 0px, better use `useOnScreen` directly which its default offset value is 0px by default.

Take a look on https://github.com/SUI-Components/adevinta-spain-components/blob/master/components/react/hooks/src/useOnScreen/index.js

Reply:
Nice! 👌 I took a quick look and didn't notice it! Thanks!